### PR TITLE
macos cached_list add instructions

### PR DESCRIPTION
**Pull request by molleweide**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -191,6 +191,37 @@ This relies on a `locate` command to find repositories. This should be much fast
 
 *Note*: at this point, the plugin does not manage index update. Updating the index often requires to run a command like `updatedb` as root.
 
+#### MacOS
```
Comment (cljoly) [inline README.md:194]: What do you think of titling this
```suggestion
#### Notes for macOS
```

File: README.md [hunk 2]
```diff
@@ -191,6 +191,37 @@ This relies on a `locate` command to find repositories. This should be much fast
...
+
+`glocate` command used for caching on macos comes with gnu `findutils` which can be installed with
+```bash
+brew install findutils
+```
+With `glocate` installed use
```
Comment (cljoly) [inline README.md:200]:
```suggestion
With `glocate` installed, add the following to your `.bashrc`/`.zshrc`
```
I suppose this needs to be added to some shell configuration, right? Which one did you use of `.bashrc` and `.zshrc`?

**Review by cljoly — COMMENTED** [review]
This is a great step forward to enhance support of macOS platforms, thanks!

I’ve commented on some minor points. We can merge with the TODO and come back to it later.

**Comment by molleweide** [comment]
thanks for reviewing clement!

**Comment by cljoly** [comment]
> Do you go by Clement or does your friends and family call you something else?

I go by Clément.

**Comment by molleweide** [comment]
> > Do you go by Clement or does your friends and family call you something else?
> 
> 
> 
> I go by Clément.

Cool, my name is Hjalmar and I live in Seeden.

**Comment by cljoly** [comment]
Nice to meet you :)

**Comment by molleweide** [comment]
Likeweise dude!

**Pull request merged**